qa/L0_batcher/test.sh

```diff
@@ -164,14 +184,22 @@ done
 rm -fr var_models && mkdir var_models
 for BACKEND in $BACKENDS; do
     TMP_MODEL_DIR="$DATADIR/qa_variable_model_repository/${BACKEND}_float32_float32_float32"
-
-    for TMP_DIR in $TMP_MODEL_DIR; do
```

Comment:
Having a for loop was not required? Does $TMP_MODEL_DIR always contain a single element?

Reply:
It did not seem like this loop was required, each of the backends only had a single entry of the format {BACKEND}_float32_float32_float32.